### DISCUSSION TITLE
Feature preview popup UI broke

### DISCUSSION THREAD

**Discussion by stephenkhoo** [community/community#6265] [Other Feature Feedback, Questions, & Ideas]
![Screenshot 2021-10-13 at 9 08 38 AM](https://user-images.githubusercontent.com/80662326/137049638-ebe7f547-f61d-4c32-9f90-b9ca4d2b2467.png)

**Comment by gaknoll** [comment]
Thanks for reporting this @stephenkhoo! I am not able to reproduce this - is it still happening? If so what OS and browser are you using?

**Reply by stephenkhoo** [reply]
Yes, it is still happening.
I'm using MacOS Big Sur 11.0.1 , and also safari 14.0.1

**Reply by stephenkhoo** [reply]
With Chrome not problem, in the same OS. So I think it is just safari's issue
